### PR TITLE
GRPC miner migration v2 - refactor

### PR DESCRIPTION
**Pull request by twiny**

### CODE REVIEW

File: buf.gen.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: v1beta1
+version: v1
```
Comment (kushthedude) [inline buf.gen.yaml:1]: Why?

File: buf.lock [hunk 2]
```diff
@@ -2,9 +2,9 @@
 version: v1
 deps:
   - remote: buf.build
-    owner: beta
+    owner: googleapis
     repository: googleapis
     branch: main
```
Comment (kushthedude) [inline buf.lock:7]: Was this auto-changed?

**Review by kushthedude — COMMENTED** [review]
Did you try to start the server on your local? Did it work?

File: buf.gen.yaml (hunk 1 above)
Comment (twiny) [reply buf.gen.yaml:1]: this is an update - v1 is a released after the v1beta1.

File: buf.lock (hunk 2 above)
Comment (twiny) [reply buf.lock:7]: Repository `buf.build/beta/googleapis` is deprecated: This module has deprecated in favor of `buf.build/googleapis/googleapis`. Please use that instead.

File: buf.gen.yaml (hunk 1 above)
Comment (kushthedude) [reply buf.gen.yaml:1]: Understood

**Pull request closed without merging**